Generators/src/main/java/io/deephaven/libs/GroovyStaticImportGenerator.java

```diff
@@ -373,15 +372,13 @@ public static void main(String[] args) throws ClassNotFoundException, IOExceptio
 
         String devroot = null;
         boolean assertNoChange = false;
-        if (args.length == 0) {
-            devroot = Configuration.getInstance().getDevRootPath();
-        } else if (args.length == 1) {
+        if (args.length == 1) {
             devroot = args[0];
         } else if (args.length == 2) {
             devroot = args[0];
             assertNoChange = Boolean.parseBoolean(args[1]);
         } else {
-            System.out.println("Usage: [<devroot> [assertNoChange]]");
+            System.out.println("Usage: <devroot> [assertNoChange]");
```

Comment:
Have you tested all of the generators?

Reply:
I've run `./gradlew Generators:generateAll` and I think testing should also assert all the generators are correct.